Mathlib/SetTheory/Cardinal/Arithmetic.lean

```diff
@@ -415,6 +415,15 @@ protected theorem ciSup_mul_ciSup (g : ι' → Cardinal.{v}) :
     (⨆ i, f i) * (⨆ j, g j) = ⨆ (i) (j), f i * g j := by
   simp_rw [Cardinal.ciSup_mul f, Cardinal.mul_ciSup g]
 
+theorem sum_eq_iSup_lift {f : ι → Cardinal.{max u v}} (hι : ℵ₀ ≤ #ι)
+    (h : lift.{v} #ι ≤ iSup f) : sum f = iSup f := by
+  apply (iSup_le_sum f).antisymm'
+  convert sum_le_iSup_lift f
+  rw [mul_eq_max (aleph0_le_lift.mpr hι) ((aleph0_le_lift.mpr hι).trans h), max_eq_right h]
+
+theorem sum_eq_iSup {f : ι → Cardinal} (hι : ℵ₀ ≤ #ι) (h : #ι ≤ iSup f) : sum f = iSup f :=
+  sum_eq_iSup_lift hι ((lift_id #ι).symm ▸ h)
```

Comment:
```suggestion
  sum_eq_iSup_lift hι (lift_id #ι ▸ h)
```
`▸` automatically uses hypothesese the right way around

Reply:
Ah darn it! I thought this was one of my PRs again. Sorry for that.

Reply:
> `▸` automatically uses hypothesese the right way around

It doesn't seem to in this case. It works when `(lift_id #ι) ▸ h` is a seperate `have` but not as part of the entire expression.

Reply:
Hmm maybe a universe unification issue